docker-hub/repos/index.md

```diff
@@ -0,0 +1,108 @@
+---
+description: Using repositories on Docker Hub
+keywords: Docker, docker, trusted, registry, accounts, plans, Dockerfile, Docker Hub, webhooks, docs, documentation, manage, repos
+title: Repositories
```

Comment:
The `Repositories` title is inconsistent with the `Manage` title in the TOC.

Reply:
Repositories or Repository? If we're referencing multiple repos?

Reply:
Thanks Craig! Updated